app/javascript/components/App.js

```diff
@@ -19,41 +19,138 @@ import UserProfileEdit from './pages/UserProfileEdit'
 import UserProfile from './pages/UserProfile'
 import MyProfileIndex from './pages/MyProfileIndex'
 
-import mockPersonalInfo from './mockPersonalInfo.js'
-import mockEmergencyContacts from './mockEmergencyContacts.js'
+// import mockPersonalInfo from './mockPersonalInfo.js'
+// import mockEmergencyContacts from './mockEmergencyContacts.js'
```

Comment:
Can we just delete instead of commenting out? We can always come back to it with Github

Reply:
Not a problem!